### ISSUE TITLE
Python 3.8 installation fails with `error: could not create 'build': File exists`

### ISSUE THREAD

**Issue by ricwo** [tensorflow/addons#1831]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): OS X
- TensorFlow version and how it was installed (source or binary): 2.2.0 (binary)
- TensorFlow-Addons version and how it was installed (source or binary): 0.9.1 (binary)
- Python version: 3.8.2
- Is GPU used? (yes/no): no

**Describe the bug**

Installation of 0.9.1 fails on py 3.8 with 

```bash
Building wheels for collected packages: tensorflow-addons
  Building wheel for tensorflow-addons (PEP 517) ... error
  ERROR: Command errored out with exit status 1:
   command: /Users/ric/.envs/x-3.8/bin/python /Users/ric/.envs/x-3.8/lib/python3.8/site-packages/pip/_vendor/pep517/_in_process.py build_wheel /var/folders/84/8zlc8yss76s95ms27w30kxk80000gn/T/tmp6qx2v39g
       cwd: /private/var/folders/84/8zlc8yss76s95ms27w30kxk80000gn/T/pip-req-build-gaukb8v2
  Complete output (5 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  error: could not create 'build': File exists
  ----------------------------------------
  ERROR: Failed building wheel for tensorflow-addons
Failed to build tensorflow-addons
ERROR: Could not build wheels for tensorflow-addons which use PEP 517 and cannot be installed directly
```

**Code to reproduce the issue**

```bash
python -V
Python 3.8.2

pip show tensorflow
Name: tensorflow
Version: 2.2.0
...
```

```bash
pip install git+https://github.com/tensorflow/addons.git@r0.9
```

**Comment by seanpmorgan** [comment]
Hi @ricwo sorry about that. We've released version 0.10 which has a published py38 build. Please give it a try and let us know if there are any issues.

**Comment by ricwo** [comment]
It works, thanks a lot 👍
